react-web/src/components/CreateCertificate/CreateCertificate.tsx

```diff
@@ -21,6 +21,25 @@ import { Address,
 import Toast from "components/Toast/Toast";
 import { fetchData } from "api/api";
 
+export interface run {
```

Comment:
only have one comment: I would remain consistent with upper case for the first letter in the case of types
same for (the `certificate`).